docker-compose.yml.dist

```diff
@@ -2,6 +2,7 @@ mail:
   image: tvial/docker-mailserver
   hostname: mail
   domainname: domain.com
+  privileged: true
```

Comment:
Ah ok. And since `privileged` is global, can we use `--cap-add NET_ADMIN` or something like that?